src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* Added meter-level tags to Prometheus exporter
```

Comment:
if this is only applicable when OpenMetrics is requested, lets mention that in the changelog too.

Additionally, it'd be nice to improve the docs too : https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.Prometheus.AspNetCore (Good candidate for a follow up PR)

Reply:
Based on the test: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5837/files#diff-ad457b7cc11412d4d6339428461e04c3938d7614391fe2f3dc72d51946002cbfR299-R303

It affects both Open Metrics and non-open metrics requests here.

Reply:
Correct, this applies to both formats